services/core-api/app/api/services/document_generator_service.py

```diff
@@ -34,10 +34,11 @@ def generate_document(cls, document_template, template_data):
             cls._push_template(document_template, fileobj)
 
         # Create the document generation request body
+        document_name_start_extra = template_data.get('document_name_start_extra')
+        document_name_start_extra = f'{document_name_start_extra} ' if document_name_start_extra else ''
         date_string = datetime.date.today().strftime("%Y-%m-%d")
```

Comment:
Should this be a UTC date?

Reply:
Good question. I'm not sure what the users are expecting. Probably the date to be in their local timezone, perhaps, but we currently don't have the user send over that information in a document generation request...

I think this is something that will be tackled in the larger datetime/timezone standardization ticket. What do you think?

Reply:
Is it easy to add user local date to generate document token? I think it might be confusing if you are looking for the document and the date of the newly generated one is tomorrow.